aws_marketplace/using_model_packages/right_sizing_amazon_sagemaker_endpoints/right-sizing-amazon-sagemaker-endpoints.ipynb

```diff
@@ -0,0 +1,1224 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "64f6469d",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# Install Locust library for load testing\n",
```

Comment:
Use the (#) for the title: "Right-sizing Amazon SageMaker Endpoints" - which you have later... so you would want to move this below the title - otherwise this line will show up as the title. You can merge this content in the prerequisites section.